### PR TITLE
QUILTME-135: Bottom pop up

### PR DESCRIPTION
**Pull request by mariannuar**
## This PR does the following:
- Adds the Bottom Pop Up

## Ticket(s)
[QUILTME-135](https://fourkitchens.clickup.com/t/36718269/QUILTME-135)

### Functional Testing
- [ ] Run `yarn install && yarn dev`
- [ ] Visit the bottom pop up banner [looks](https://www.figma.com/file/cHkcphD8wHi58n48kR9r6c/QuiltMe.ai?type=design&node-id=252-380&t=cfwH9Df3pOhZCGyH-4) and [works](https://www.figma.com/proto/cHkcphD8wHi58n48kR9r6c/QuiltMe.ai?type=design&node-id=245-162&scaling=min-zoom&page-id=245%3A161&starting-point-node-id=341%3A114) as expected

### CODE REVIEW

File: src/components/ui/02-molecules/bottom-banner.jsx [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+                  >
+                    Let’s Talk
+                  </Button>
+                </Stack>
+              </Grid>
+            </Grid>
+            <Box sx={creditsContainerStyles}>
+              <Typography variant="small-text" sx={creditsTextStyles}>
+                Credits: Adam Erickson, Ashley Hitson, Hector Lopez, Joanna Cendrowski, Katy Fernández, 
+                Kelvin Cheng, Laura Johnson, Mari Núñez, Mike Goulding, Randy Dean West, Roberto Hernández,
```
Comment (amazingrando) [inline src/components/ui/02-molecules/bottom-banner.jsx]: Please change "West" to "Oest". Thanks! (I'm used to this happening with autocorrect.)

**Review by amazingrando — APPROVED** [review]
Code review approved. Do not merge until I do a functional review.

**Comment by amazingrando** [comment]
@mariannuar I did a functional review and I'm not sure that using a modal is the right way to go for revealing the content.

I've recorded a loom to walk you through my thoughts at https://www.loom.com/share/0e3b71e34bfd4d049dd4216357f2d18f.

Highlights:
* We need to have the content reveal more like what [is in the prototype](https://www.figma.com/proto/cHkcphD8wHi58n48kR9r6c/QuiltMe.ai?page-id=245%3A161&type=design&node-id=245-162&viewport=693%2C485%2C0.39&scaling=scale-down&starting-point-node-id=341%3A114&show-proto-sidebar=1). I'm okay if it doesn't perfectly match but we have to get closer.
* You should consider using the [Drawer component](https://mui.com/material-ui/react-drawer/). Its reveal animation and setup is a lot like what is in the prototype.

File: src/components/ui/02-molecules/bottom-banner.jsx (hunk 1 above)
Comment (mariannuar) [reply src/components/ui/02-molecules/bottom-banner.jsx]: Omg I'm really sorry! Fixed

**Comment by mariannuar** [comment]
@amazingrando This is fixed, let me know what you think! Btw having the same behaviour for the prototype was kind of difficult, I even tried to do it with the Drawer component, but I couldn't get it. Thank you for your feedback!

**Comment by mariannuar** [comment]
I have to fix some things based on Ashley's feedback @amazingrando . I'll be fixing those today. Adding the `Needs work` label. Thanks!

**Review by amazingrando — APPROVED** [review]
Approved!

**Pull request merged**